docs/mkdocs.yml

```diff
@@ -251,23 +251,21 @@ nav:
         - how-tos/http/custom_routes.md
       - Deployment options:
         - Overview: concepts/deployment_options.md
-        - concepts/platform_architecture.md
-        - Scalability & resilience: concepts/scalability_and_resilience.md
+        - LangGraph Data Plane: concepts/langgraph_data_plane.md
+        - LangGraph Control Plane: concepts/langgraph_control_plane.md
         - Cloud SaaS:
           - Overview: concepts/langgraph_cloud.md
           - Deploy Cloud SaaS: cloud/deployment/cloud.md
         - Self-Hosted Data Plane:
           - Overview: concepts/langgraph_self_hosted_data_plane.md
-          - LangGraph Data Plane: concepts/langgraph_data_plane.md
           - Deploy Self-Hosted Data Plane: cloud/deployment/self_hosted_data_plane.md
         - Self-Hosted Control Plane:
           - Overview: concepts/langgraph_self_hosted_control_plane.md
-          - LangGraph Control Plane: concepts/langgraph_control_plane.md
           - Deploy Self-Hosted Control Plane: cloud/deployment/self_hosted_control_plane.md
         - Standalone Container:
           - Overview: concepts/langgraph_standalone_container.md
           - Deploy Standalone Container: cloud/deployment/standalone_container.md
-          # ADD SH LITE?
```

Comment:
I think we're missing information about Self-Hosted Lite, wdyt? If it doesn't warrant its own page, I think there should be a heading for it on the Standalone Container overview page

Reply:
I'll add a heading for it in the `Standalone Container` page. Will clarify offline.

Reply:
Actually, I'm going to do this in a separate, smaller PR.